Source/WebCore/Modules/mediastream/MediaStreamTrack.h

```diff
@@ -149,8 +149,8 @@ class MediaStreamTrack
 
     RealtimeMediaSource& source() const { return m_private->source(); }
     RealtimeMediaSource& sourceForProcessor() const { return m_private->sourceForProcessor(); }
-    MediaStreamTrackPrivate& privateTrack() { return m_private.get(); }
-    const MediaStreamTrackPrivate& privateTrack() const { return m_private.get(); }
+    Ref<MediaStreamTrackPrivate> privateTrack() { return m_private; }
```

Comment:
This is going against the new guidelines. We should keep the old getters that return references as they're still useful for calling trivial functions. And add new a `protectedTrack()` getter instead which returns a smart pointer.

Reply:
But also, does `m_private` ever get reassigned? If not, you can just mark the data member as `const` and you won't need a `protectedTrack()` getter at all.